README.md

```diff
@@ -1,60 +1,36 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+Project Name: Dario Alessio's Portafolio
 
-> Description the project.
+Description of project: 
 
+This is a page for mobile devices create to include professional information regarding technical and soft skills and abilities to inform potential employers.
 
-## Built With
+This is the first part of the whole project and so it includes only the homepage for mobile devices.
 
-- Major languages
-- Frameworks
-- Technologies used
+Project will be updated in next weeks with more information about the completed projects that will form part of the portafolio.
 
-## Live Demo (if available)
+The project hs been done following template # 4 of Figma options provided by Microverse.
```

Comment:
As the previous reviewer asked you, you should only customize it by adding your information so please try to leave all the important header from the[ template README](https://github.com/microverseinc/readme-template) and add you information for every important header